### ISSUE TITLE
Improper border around page area box

### ISSUE THREAD

**Issue by zopyx** [Kozea/WeasyPrint#1053]
Check attached sample (WeasyPrint 51)

The image in the head page area box should be surrounded by a green.
Instead the box border is drawn through the image.

https://www.dropbox.com/s/xt6pc79kwqnx5ev/wp.zip?dl=0

**Comment by liZe** [comment]
I’m not sure that the image should be surrounded by a green border. Browsers (even if they don’t support `running`) also draw the border through the image, because the span is inline.

But the missing left border is a problem for sure.

**Comment by liZe** [comment]
The problem is caused by the space before the image. This leading space should be removed, but it’s not. The span is split into 2 lines, the first one with the space, the second one with the image.

**Comment by liZe** [comment]
@zopyx Do you have a copy of the original sample? It’s been removed from Dropbox and I can’t remember how we can reproduce the bug.

**Comment by zopyx** [comment]
Sorry, too long ago :)

**Comment by liZe** [comment]
I’ve found a way to reproduce, and it’s not fixed yet…

```html
<style>
  @page {
    @top-center {
      content: element(heading);
    }
  }
  span {
    border: 5px solid pink;
    position: running(heading);
  }
  img {
    width: 1cm;
  }
</style>
abc
<span> <img src="file:///home/lize/Images/Burst.jpg" /></span>
```

**Comment by liZe** [comment]
It’s been fixed in version 57.0.
